packages/react/src/components/form/Fieldset/Fieldset.stories.tsx

```diff
@@ -9,4 +11,23 @@ export default {
   component: Fieldset,
 } as Meta;
 
-export const Preview: Story = (args) => <Fieldset {...args}></Fieldset>;
+export const Preview: Story = (args) => (
+  <form>
+    <Fieldset {...args}>
+      <RadioGroup legend=''>
```

Comment:
This `RadioGroup` is redundant here as its the [same `Fieldset` component](https://github.com/digdir/designsystemet/blob/20278fa5727e6ea1c3e2867835f1a31b23f86bc9/packages/react/src/components/form/Radio/Group/Group.tsx#L57) just with some additional props for handling radios in groups, such as `name`.

Reply:
Its probably due to them using the same React context, since its passed there to all child components, which includes all components under `components/form`